### PR TITLE
docimports for painting

### PR DESCRIPTION
**Pull request by goderbauer**
Part of https://github.com/flutter/flutter/issues/150800

### CODE REVIEW

File: packages/flutter/lib/src/painting/basic_types.dart [hunk 1]
```diff
@@ -2,6 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// @docImport 'package:flutter/rendering.dart';
+/// @docImport 'package:flutter/material.dart';
```
Comment (srawlins) [inline packages/flutter/lib/src/painting/basic_types.dart:6]: Do you sort these? We don't'\ have a lint yet 😛

File: packages/flutter/lib/src/painting/image_provider.dart [hunk 2]
```diff
@@ -15,10 +18,10 @@ import 'binding.dart';
 import 'image_cache.dart';
 import 'image_stream.dart';
 
-/// Signature for the callback taken by [_createErrorHandlerAndKey].
+/// Signature for the callback taken by [ImageProvider._createErrorHandlerAndKey].
```
Comment (srawlins) [inline packages/flutter/lib/src/painting/image_provider.dart:21]: Excellent

**Review by srawlins — APPROVED** [review]

File: packages/flutter/lib/src/painting/basic_types.dart (hunk 1 above)
Comment (goderbauer) [reply packages/flutter/lib/src/painting/basic_types.dart:6]: I tried to manually sort these. Looks like messed up my ABCs here. Will fix - and cannot wait for the lint!

**Comment by matanlurey** [comment]
test-exempt: `@docImport` is code-comments only.

**Pull request merged**